testing/kfctl/kf_is_ready_test.py

```diff
@@ -145,6 +145,34 @@ def test_kf_is_ready(namespace, use_basic_auth, use_istio, app_path):
     logging.info("Verifying that deployment %s started...", deployment_name)
     util.wait_for_deployment(api_client, knative_namespace, deployment_name, 10)
 
+  if platform == "gcp":
```

Comment:
Use pytest.Skip to skip the test if we aren't running on GCP
http://doc.pytest.org/en/latest/skipping.html